src/api/app/models/workflow/step/configure_repositories.rb

```diff
@@ -1,8 +1,10 @@
 class Workflow::Step::ConfigureRepositories < Workflow::Step
   REQUIRED_KEYS = [:project, :repositories].freeze
-  REQUIRED_REPOSITORY_KEYS = [:architectures, :name, :target_project, :target_repository].freeze
+  REQUIRED_REPOSITORY_KEYS = [:architectures, :name, :paths].freeze
+  REQUIRED_REPOSITORY_PATH_KEYS = [:target_project, :target_repository].freeze
```

Comment:
💭  This naming is a bit confusing as there are repository paths and yaml paths now...

Reply:
The keys under `paths` refer to repository paths, so I don't see the confusion here. They refer to the same concept.